<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryDialog</class>
 <widget class="QDialog" name="HistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Historial</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <property name="spacing">
    <number>16</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Historial</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="headerSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="filtersCard">
     <layout class="QVBoxLayout" name="filtersLayout">
      <property name="leftMargin">
       <number>22</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="rightMargin">
       <number>22</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QLabel" name="hintLabel">
        <property name="text">
         <string>Filtra por fechas para consultar tus sesiones.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="filterFormLayout">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignVCenter</set>
        </property>
        <property name="horizontalSpacing">
         <number>16</number>
        </property>
        <property name="verticalSpacing">
         <number>12</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="fromLabel">
          <property name="text">
           <string>Desde:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="fromDateEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="toLabel">
          <property name="text">
           <string>Hasta:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="toDateEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="filterButtonsLayout">
        <item>
         <spacer name="filterButtonsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="applyButton">
          <property name="text">
           <string>Aplicar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="resultsCard">
     <layout class="QVBoxLayout" name="resultsLayout">
      <property name="leftMargin">
       <number>22</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="rightMargin">
       <number>22</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QTableWidget" name="sessionsTable"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
   <widget class="QFrame" name="totalsCard">
    <layout class="QVBoxLayout" name="totalsLayout">
      <property name="leftMargin">
       <number>22</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="rightMargin">
       <number>22</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="summaryTitleLabel">
        <property name="text">
         <string>Resumen</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="statsLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QFrame" name="sessionsStatFrame">
          <layout class="QVBoxLayout" name="sessionsStatLayout">
           <property name="leftMargin">
            <number>14</number>
           </property>
           <property name="topMargin">
            <number>12</number>
           </property>
           <property name="rightMargin">
            <number>14</number>
           </property>
           <property name="bottomMargin">
            <number>12</number>
           </property>
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="sessionsValueLabel">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sessionsLabel">
             <property name="text">
              <string>Sesiones</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="hitsStatFrame">
          <layout class="QVBoxLayout" name="hitsStatLayout">
           <property name="leftMargin">
            <number>14</number>
           </property>
           <property name="topMargin">
            <number>12</number>
           </property>
           <property name="rightMargin">
            <number>14</number>
           </property>
           <property name="bottomMargin">
            <number>12</number>
           </property>
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="hitsValueLabel">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="hitsLabel">
             <property name="text">
              <string>Aciertos</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="faultsStatFrame">
          <layout class="QVBoxLayout" name="faultsStatLayout">
           <property name="leftMargin">
            <number>14</number>
           </property>
           <property name="topMargin">
            <number>12</number>
           </property>
           <property name="rightMargin">
            <number>14</number>
           </property>
           <property name="bottomMargin">
            <number>12</number>
           </property>
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="faultsValueLabel">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="faultsLabel">
             <property name="text">
              <string>Fallos</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomButtonsLayout">
     <item>
      <spacer name="bottomButtonsSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
